src/gui/mainwindow.h

```diff
@@ -103,7 +103,7 @@ class MainWindow: public QMainWindow
     void showNotificationBaloon(QString title, QString msg) const;
 
 private slots:
-    void toggleVisibility(QSystemTrayIcon::ActivationReason e = QSystemTrayIcon::Trigger);
+    void toggleVisibility(QSystemTrayIcon::ActivationReason reason = QSystemTrayIcon::Trigger);
```

Comment:
I said it wrong. I was referring to simple types (int, bool, double etc.) the size of which does not exceed the size of the pointer.